lib/robots/dor_repo/gis_assembly/generate_content_metadata.rb

```diff
@@ -38,6 +38,9 @@ def perform_work
           geo_data_xml = doc.dup.xpath('/mods:mods/mods:extension[@displayLabel="geo"]/rdf:RDF/rdf:Description', ns).first
 
           xml = create_content_metadata(objects, geo_data_xml)
+          # metadata dir was not created upstream because no more writing geoMetadata.xml or descMetadata.xml
```

Comment:
Had to add this back in to get the integration to pass, but I assume it will get removed once this step is refactored.